### PR TITLE
Convert publication description to templ

### PR DESCRIPTION
**Pull request by nics**
- templ fmt all views
- replace publication show_description and all it's dependent templates
- more concise url builder methods (still have to replace the old ones)
- most of the old templates are still in use by other parts of the app, so haven't deleted those yet

### CODE REVIEW

File: handlers/publicationviewing/show.go [hunk 1]
```diff
@@ -21,14 +21,14 @@ type YieldShow struct {
...
-	DisplayAdditionalInfo *display.Display
-}
+// type YieldShowDescription struct {
+// 	Context
+// 	SubNavs               []string
+// 	ActiveSubNav          string
+// 	DisplayDetails        *display.Display
+// 	DisplayConference     *display.Display
+// 	DisplayAdditionalInfo *display.Display
+// }
```
Comment (verheyenkoen) [inline handlers/publicationviewing/show.go]: Forgot to delete this?

File: handlers/publicationviewing/show.go [hunk 2]
```diff
@@ -71,15 +71,19 @@ func (h *Handler) Show(w http.ResponseWriter, r *http.Request, ctx Context) {
...
+	redirectURL := c.PathTo("publications").String()
+	publicationviews.Description(c, p, redirectURL).Render(r.Context(), w)
+	// render.View(w, "publication/show_description", YieldShowDescription{
+	// 	Context:               ctx,
+	// 	SubNavs:               subNavs,
+	// 	ActiveSubNav:          "description",
+	// 	DisplayDetails:        displays.PublicationDetails(ctx.User, ctx.Loc, ctx.Publication),
+	// 	DisplayConference:     displays.PublicationConference(ctx.User, ctx.Loc, ctx.Publication),
+	// 	DisplayAdditionalInfo: displays.PublicationAdditionalInfo(ctx.User, ctx.Loc, ctx.Publication),
+	// })
```
Comment (verheyenkoen) [inline handlers/publicationviewing/show.go]: Forgot to delete this?

File: views/display/field.templ [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	</div>
+}
+
+// templ String(val string) {
+//     if val != "" {
+//         <span>{ val }</span>
+//     } else {
+//         <span>-</span>
+//     }
+// }
```
Comment (verheyenkoen) [inline views/display/field.templ:43]: Forgot to delete this comment?

**Review by verheyenkoen — COMMENTED** [review]
Marked some leftover comments or were these intended?

File: views/display/field.templ (hunk 3 above)
Comment (nics) [reply views/display/field.templ]: deleted

**Review by verheyenkoen — APPROVED** [review]

**Pull request merged**